something/MyPhpClass.php

```
@@ -0,0 +1,13 @@
+<?php
+
+class MyPhpClass
+{
+    private string $myProperty;
+
+    public function contactParams(string $p1, string $p2, string $p3, string $p4, string $p5): string
```

PHPMD (2.15.0)

Avoid variables with short names like $p1. Configured minimum length is 3. (Naming Rules)

(ShortVariable)